src/tools/utils.js

```diff
@@ -4,18 +4,32 @@ import dotProp from 'dot-prop';
 import _ from 'lodash';
 import log from './logger';
 
+export function keywordArrayReplace(input, mappings) {
+  Object.keys(mappings).forEach(function(key) {
+    // Matching against two sets of patterns because a developer may provide their array replacement keyword with or without wrapping quotes. It is not obvious to the developer which to do depending if they're operating in YAML or JSON.
+    const pattern = `@@${key}@@`;
+    const patternWithQuotes = `"${pattern}"`;
```

Comment:
The key functional change in this PR is using regex to replace the `@@` pattern with and without wrapping quotes.